dipy/workflows/tracking.py

```diff
@@ -18,6 +20,58 @@
 from dipy.data import default_sphere
 from dipy.direction import DeterministicMaximumDirectionGetter
 from dipy.workflows.multi_io import io_iterator_
+from ipdb import set_trace
+
+
+def det_track_flow(peaks_files, stopping_files, use_sh=False,
+                   out_dir='', out_tractogram='tractogram.trk'):
+
+    """ Workflow for deterministic tracking
+
+    Parameters
+    ----------
+    peaks_files : string
+       Path to the peaks values files. This path may contain wildcards to use
+       multiple masks at once.
+    stopping_files : string
+        Path of FA or other images used for stopping criteria for tracking.
+    use_sh : bool
+        Use spherical harmonics saved in peask to find the maximum peak cone.
```

Comment:
peaks or peaks_files?

so peaks_files can be a sh_files? maybe better naming it tracking_files?